docs/core/linux-prerequisites.md

```diff
@@ -114,7 +136,19 @@ To install .NET Core on supported RHEL versions:
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-To ensure you have the latest installation information, follow the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-current) for supported RHEL versions.
+**.NET Core 2.0**
+
+ Install .NET Core 2.x on supported RHEL versions:
+
+* .NET Core Runtime 2.0.8 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.8)
+* .NET Core Runtime 2.0.7 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.7)
+* .NET Core Runtime 2.0.6 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.6)
+* .NET Core Runtime 2.0.5 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/runtime-2.0.5)
+* .NET Core SDK 2.1.200 [install link](https://www.microsoft.com/net/download/linux-package-manager/rhel/sdk-2.1.200)
```

Comment:
These are all still 2.0 SDKs. Might want to call them out like they do in the .net site to make it less confusing (screenshot from https://www.microsoft.com/net/download/all):

![image](https://user-images.githubusercontent.com/12971179/40867316-1fd00ba2-65b8-11e8-92b7-fe23efc46616.png)